test-vectors/presentation_exchange/README.md

```diff
@@ -8,15 +8,16 @@ Input and output for a full presentation exchange test vectors are available [he
 
 the value of `input` is a an object with `presentationDefinition` and the corresponding `credentialJwt`
 
-| Property                | Description                                                                                                                                                                                |
-| ----------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
-| `presentationDefinition`           | the input [presentationDefinition](https://identity.foundation/presentation-exchange/#presentation-definition)  showing the requirements used for this getting an example WA license               |
-| `credentialJwt`   | the input [verifiable credential secured as a JWT](https://www.w3.org/TR/vc-data-model/#json-web-token) that corresponds to the presentationDefinition to fulfill it and do a full presentation exchange
+| Property                 | Description                                                                                                                                                                                              |
+| ------------------------ | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `presentationDefinition` | the input [presentationDefinition](https://identity.foundation/presentation-exchange/#presentation-definition) showing the requirements used for this getting an example WA license                      |
+| `credentialJwt`          | the input [verifiable credential secured as a JWT](https://www.w3.org/TR/vc-data-model/#json-web-token) that corresponds to the presentationDefinition to fulfill it and do a full presentation exchange |
 
 ### Output
 
-the value of `output` is an object that contains the following properties
+the value of `output` is an object that may contain one of the following properties.
```

Comment:
this language weakens the requirement. what is testable if the properties are may?

Reply:
This section is explaining what's included in the output  object in a vector object - some test vector output only includes the presentationSubmission object and some others (like the select-credentials one in this PR) only include selectedCredentials array. 

I could say "one of" instead of "may include" to make it a bit more clear. 

I don't think this weakens our test requirement as it's just wording around describing the test vector object shapes.

Reply:
@decentralgabe @jiyoontbd To avoid deciding too much whether "may" is the right word, I pushed a commit adding much more detail to this README that describes shape of each set of vectors separately.